Add more deprecated/renamed plugins to exclude from the Update Centre.

orrc opened the pull request:
Related to #14, I went through the entire plugin list and went searching through artifact IDs, GitHub repos, old wiki pages and ended up with this list of (mostly) unreported artifact ID renames.

I still have a few more to look into more closely...

daniel-beck commented:
:+1:

evernat commented:
I suggest to also ignore following artifact ids of renamed plugins:
- jenkinswalldisplay-pom ([renamed] (https://github.com/jenkinsci/walldisplay-plugin/commit/ec2600d7e2b65c04b570bf8ddad88e3ed17736f0#diff-600376dffeb79835ede4a0b285078036) to jenkinswalldisplay)
- perfectomobile-jenkins (renamed to perfectomobile)
- paaslane ([renamed] (https://github.com/jenkinsci/paaslane-plugin/commit/ba83d53b0d038ab2153f9c6c64f140c416c53c1f#diff-600376dffeb79835ede4a0b285078036) to paaslane-estimate)
- sms-notification ([renamed] (https://github.com/jenkinsci/sms-plugin/commit/728be3a932d032a89f3673075e6764084325a6ef#diff-600376dffeb79835ede4a0b285078036) to sms)

orrc commented:
@evernat Thanks! I double-checked those plugins and added them to this commit.
@daniel-beck I added myself to this repo via the IRC bot, but still don't have write access... could you merge, please?

daniel-beck commented:
@orrc Could you please update this to current master?

orrc commented:
@daniel-beck Rebased.

daniel-beck commented:
@orrc Merged, thanks a lot!
